araproc/analysis/standard_reco.py

```diff
@@ -5,6 +5,7 @@
 import ROOT
 from scipy.interpolate import Akima1DInterpolator
 import itertools
+import time
```

Comment:
Maybe we should drop this now that it's not being used?

Reply:
Yup, will do